Package.swift

```diff
@@ -0,0 +1,41 @@
+// swift-tools-version: 6.0
+
+import PackageDescription
+
+let package = Package(
+    name: "swift-collections-differentiable",
+    products: [
+        .library(
+            name: "CollectionsDifferentiable",
+            targets: ["CollectionsDifferentiable"]
+        ),
+        .library(
+            name: "OrderedCollectionsDifferentiable",
+            targets: ["OrderedCollectionsDifferentiable"]
+        ),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/apple/swift-collections.git", from: "1.1.4"),
+    ],
+    targets: [
+        .target(
+            name: "CollectionsDifferentiable",
+            dependencies: [
+                .product(name: "Collections", package: "swift-collections"),
+                "OrderedCollectionsDifferentiable"
+            ]
+        ),
+        .target(
+            name: "OrderedCollectionsDifferentiable",
+            dependencies: [
+                .product(name: "OrderedCollections", package: "swift-collections"),
+            ]
+        ),
+        .testTarget(
+            name: "OrderedCollectionsDifferentiableTests",
+            dependencies: [
+                "OrderedCollectionsDifferentiable",
+            ]
```

Comment:
looks like we1re missing a `CollectionsDifferentiable` `testTarget`

Reply:
Right now I went for just a single target since `CollectionsDifferentiable` just exports all the other modules and doesn't contain any implementation. We might add more once we have extensions for more modules.